tests/store/artifact/test_databricks_model_artifact_repo.py

```diff
@@ -0,0 +1,151 @@
+            assert artifacts[0].is_dir is False
+            assert artifacts[0].file_size == 294
+            assert artifacts[1].path == "data"
+            assert artifacts[1].is_dir is True
+            assert artifacts[1].file_size is None
+            call_endpoint_mock.assert_called_with(ANY, REGISTRY_LIST_ARTIFACTS_ENDPOINT)
+
+    def test_list_artifacts_for_single_file(self, databricks_model_artifact_repo):
+        list_artifact_file_response_mock = mock.MagicMock
+        list_artifact_file_response_mock.status_code = 200
+        list_artifact_file_json_mock = {
+            "files": [{"path": "MLmodel", "is_dir": False, "file_size": 294}]
+        }
+        list_artifact_file_response_mock.text = json.dumps(list_artifact_file_json_mock)
+        with mock.patch(
+            DATABRICKS_MODEL_ARTIFACT_REPOSITORY + "._call_endpoint"
+        ) as call_endpoint_mock:
+            # Calling list_artifacts() on a path that's a file should return an empty list
+            call_endpoint_mock.return_value = list_artifact_file_response_mock
+            artifacts = databricks_model_artifact_repo.list_artifacts("MLmodel")
+            assert len(artifacts) == 0
+
+    @pytest.mark.parametrize(
+        "remote_file_path, local_path",
+        [
+            ("test_file.txt", ""),
+            ("test_file.txt", None),
+            ("output/test_file", None),
+            ("test_file.txt", ""),
+        ],
+    )
+    def test_databricks_download_file(
+        self, databricks_model_artifact_repo, remote_file_path, local_path
+    ):
+        signed_uri_response_mock = mock.MagicMock
+        signed_uri_response_mock.status_code = 200
+        signed_uri_mock = {
+            "signed_uri": "https://my-amazing-signed-uri-to-rule-them-all.com/1234-numbers-yay-567"
+        }
+        signed_uri_response_mock.text = json.dumps(signed_uri_mock)
+        with mock.patch(
+            DATABRICKS_MODEL_ARTIFACT_REPOSITORY + "._call_endpoint"
+        ) as call_endpoint_mock, mock.patch(
+            DATABRICKS_MODEL_ARTIFACT_REPOSITORY_PACKAGE + ".download_file_using_signed_uri"
+        ) as download_mock:
+            call_endpoint_mock.return_value = signed_uri_response_mock
+            download_mock.return_value = None
+            databricks_model_artifact_repo.download_artifacts(remote_file_path, local_path)
+            call_endpoint_mock.assert_called_with(ANY, REGISTRY_ARTIFACT_PRESIGNED_URI_ENDPOINT)
+            download_mock.assert_called_with(signed_uri_mock["signed_uri"], ANY, ANY)
+
+    def test_databricks_download_file_get_request_fail(self, databricks_model_artifact_repo):
+        with mock.patch(
+            DATABRICKS_MODEL_ARTIFACT_REPOSITORY + "._call_endpoint"
+        ) as call_endpoint_mock:
+            call_endpoint_mock.side_effect = MlflowException("MOCK ERROR")
+            with pytest.raises(MlflowException):
+                databricks_model_artifact_repo.download_artifacts("Something")
```

Comment:
add a simple test to validate that an error is thrown when calling log_artifact, log_artifacts, delete_artifact so we have code coverage